Angular2WebpackVisualStudio/src/Angular2WebpackVisualStudio/angular2App/vendor.ts

```diff
@@ -6,3 +6,17 @@ import 'bootstrap/dist/css/bootstrap-theme.css';
 
 import '../favicon.ico';
 
```

Comment:
@afnpires First of all, thanks a million for your pull request!
This causes the production size bundles to increase in size. Could you revert this.